bool hasFace
bool hasHead
bool isMultiface
sensor_msgs/Image FaceImage


